templates/common/_base/files/sysctl-icmp-conf.yaml

```diff
@@ -0,0 +1,6 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/sysctl.d/icmp.conf"
+contents:
+  inline: |
+    net.ipv4.icmp_ratemask = {{ .icmpRateMask }} 
```

Comment:
That's...going to have weird semantics.  This file ends up getting rendered based on the dynamic contents of the host that happens to be running the MCC.

It's probably simplest to just write a fixed value for now.

If we really can't write a fixed value, then this is going to need to be special cased in the MCD or so.

Reply:
I can set it to 6180 which will be correct for *our* SDNs, not quite sure about third parties. @danwinship any inputs here?

Reply:
I don't mean to bounce this around (and I know I mentioned the MCO in the other PR) but I hadn't realized that the value needed to be *dynamic* in that sense - the MCO templates just don't work that way.

Isn't there a container that's run as part of the SDN that could do the equivalent of https://github.com/openshift/sdn/pull/135 but would run regardless of the SDN type?  Just an init container or something?

Reply:
It's not that it needs to be dynamic, it's just that in theory, someone might be setting this sysctl for some other reason, and so one of us would end up stomping on the other one now. Whatever. Just give it an early name ("00-icmp.conf" or whatever) and if someone else wants to overwrite our value, let them.

Reply:
or do it via a script that can do "get/modify/write" rather than just "write"

Reply:
> or do it via a script that can do "get/modify/write" rather than just "write"

Right, so if we want to that path we can inject a systemd unit, but that then gets into "OK now how do I run code from the MCO" which is a messy topic; we have mostly been moving this type of stuff to pull containers from elsewhere.

Which goes to my comment earlier - the SDN runs as a daemonset already, isn't there a variant of openshift/sdn#135 that would run everywhere?

> Whatever. Just give it an early name ("00-icmp.conf" or whatever) and if someone else wants to overwrite our value, let them.

That also seems sane to me.

Reply:
> Which goes to my comment earlier - the SDN runs as a daemonset already, isn't there a 
variant of openshift/sdn#135 that would run everywhere?

We don't have a daemonset that would run on everywhere for every cluster, we'd need to patch it on kuryr or OVN as well, or add yet another daemonset.

> That also seems sane to me.
Does it seem sane to you in its current state? Note it's been modified to have a static value.

Reply:
> the SDN runs as a daemonset already, isn't there a variant of [openshift/sdn#135](https://github.com/openshift/sdn/pull/135) that would run everywhere?

(No part of https://github.com/openshift/sdn/ is every deployed into any cluster that is not `networkType: OpenShiftSDN`. Cluster-network-operator could do something but we don't currently have any daemonset that is deployed in 100% of configurations.)